driver/compile_common.ml

```diff
@@ -16,17 +16,8 @@
 open Misc
 
 type info = {
-<<<<<<< HEAD
-  source_file : string;
-  module_name : Compilation_unit.t;
-  output_prefix : string;
-||||||| 121bedcfd2
-  source_file : string;
-  module_name : string;
-  output_prefix : string;
-=======
   target: Unit_info.t;
->>>>>>> 5.2.0
+  module_name : Compilation_unit.t;
```

Comment:
This should probably replace `modname` in `Unit_info.t` and be removed from here?

Reply:
Yes, but in due course.  See https://github.com/ocaml-flambda/flambda-backend/wiki/Strategy-for-5.2.0-Unit_info-and-compunit

Reply:
OK.  I worry a little bit that having `modname` sitting over there could re-introduce whatever issues `Compilation_unit.t` was introduced to solve, but I defer to you and Luke here.